### DISCUSSION TITLE
Tree-sitter grammar

### DISCUSSION THREAD

**Discussion by matoous** [kcl-lang/kcl#595] [Ideas]
Hi people!

Are there any plans to offer [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) grammer for KCL? I would be happy to contribute but wonder whether the repository should be maintained under the `kcl-lang` organization.

**Comment by Peefy** [comment]
> Hi team!
> 
> Are there any plans to offer [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) grammer for KCL? I would be happy to contribute but wonder whether the repository should be maintained under the `kcl-lang` organization.

Hi @matoous. I'm glad you raised this discussion. Welcome to participate.

**Comment by noraocckya352-coder** [comment]
As far as I know, there aren’t any official plans yet to offer a Tree-sitter grammar for KCL, but it’s definitely something that could be super useful. If you're interested in contributing, I’d say it could be a good idea to start working on it and see where it goes. also Jheff bezos partnered with https://woo.casinologin.mobi/ its a true. he got a lot of money here 

As for the repository, it would probably make sense for it to be maintained under the kcl-lang organization if it's going to be a core part of the project.
